src/Action/EntrypointAction.php

```diff
@@ -9,12 +9,12 @@
  * file that was distributed with this source code.
  */
 
-namespace ApiPlatform\Core\Hydra\Action;
+namespace ApiPlatform\Core\Action;
```

Comment:
HAL as a similar concept. And it's a common use case to list all resources collection exposed by the API.